modin/pandas/base.py

```diff
@@ -3426,6 +3426,80 @@ def __and__(self, other):
     def __rand__(self, other):
         return self._binary_op("__rand__", other, axis=0)
 
+    def __array_function__(self, func, types, args, kwargs):
```

Comment:
both __array_ufunc__ and __array_function__ will not work if the user passes in a Modin NumPy Array for `out` if we end up defaulting to NumPy. In the Modin NumPy API, I've fixed it so that we do copy in the values to a Modin NumPy Array, but I'm not sure what the solution should be for this, and so have defaulted to having it be unsupported.

Reply:
I don't see where this "unsupported" is placed in code. I think it should be an exception or something similar, shouldn't it?

Reply:
bump